recipes/qt/6.x.x/conanfile.py

```diff
@@ -384,6 +397,7 @@ def _configure_cmake(self):
 
         self._cmake.definitions["INSTALL_MKSPECSDIR"] = os.path.join(self.package_folder, "res", "archdatadir", "mkspecs")
         self._cmake.definitions["INSTALL_ARCHDATADIR"] = os.path.join(self.package_folder, "res", "archdatadir")
+        self._cmake.definitions["INSTALL_LIBEXECDIR"] = os.path.join(self.package_folder, "res", "archdatadir", "bin" if self.settings.os == "Windows" else "libexec")
```

Comment:
Whats the motivation behind this change?

Reply:
it mimics this https://github.com/qt/qtbase/blob/4b60cea6020c99dabb3b2777e30f0a3924dcd8a7/src/corelib/global/qlibraryinfo.cpp#L320-L324 and this https://github.com/qt/qtbase/blob/4b09522c23e9efdf83ba8d4af436d8a700ccb66e/cmake/QtBuild.cmake#L112-L116
This folder is the one storing moc and other executables in qt 6.1.0

Reply:
Is it only for >=6.1.0?

Reply:
well, with this PR, it is for all versions.